src/App.jsx

```diff
@@ -0,0 +1,48 @@
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+import Places from './components/places/Places'
+import NotFound from './components/utils/NotFound'
+import Register from './components/register_login_forms/Register'
+import Login from './components/register_login_forms/Login'
+import NoticeAlert from './components/utils/NoticeAlert'
+import ProtectedRoute from './components/utils/ProtectedRoute'
+import PublicRoute from './components/utils/PublicRoutes'
+import PlaceForm from './components/place_form/PlaceForm.jsx'
+import NavBar from './components/navigation_bar/NavBar'
+import './App.css'
+import ReservationForm from './components/reservation_form/ReservationForm'
+import MyReservations from './components/my_reservations/MyReservations.jsx'
+import PlaceDetails from './components/place_details/PlaceDetails.jsx'
+import MyPlaces from './components/my_places/MyPlaces.jsx'
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

Reply:
Hello!
Thanks for your feedback 🙌🏼 
This is solved 😄